style.css

```diff
@@ -1 +1,218 @@
 /*  Add your styling here */
+
+:root {
+  --light-pink: rgb(247, 229, 239);
+  --light-orange: rgb(243, 239, 237);
+  ;
+}
+
+.grid-container {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-auto-rows: minmax(200px, auto); 
+  grid-gap: 1rem;
+}
+
+header {
+  grid-column: 1 / -1;
+  display: flex;
+  justify-content: space-between;
+  background-color: var(--light-orange);
+}
+
+.company-logo-img {
+  width: 140px;
+  height: 140px;
+}
+
+.nav-bar {
+  display: none;
+}
+
+.hamburger-motto {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: flex-end;
+  text-align: right;
+  padding: 20px 20px 0 0;
+}
+
+.bar {
+display: block;
+width: 35px;
+height: 4px;
+margin: 5px auto;
+background-color: #101010;
+}
+
+main {
+  grid-column: 1 / -1;
+  padding: 15px;
+  background-color: var(--light-pink);
+}
+
+.welcome {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    grid-gap: 1rem;
+    grid-auto-rows: auto;
+    margin-bottom: 30px;
+    grid-template-areas:
+      "welcome welcome"
+      "image image"
+}
+
+.welcome-text {
+  grid-area: welcome;
+  text-align: right;
+}
+
+.welcome img {
+  grid-area: image;
+  /* height: 200px; */
+}
+
+.products {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: 200px 200px;
+    grid-gap: 1rem;
+}
+
+  img {
+  width: 100%;
+  height: 200px;
+  object-fit: cover;
+}
+
+footer {
+  grid-column: 1 / -1;
+  display:grid;
+  background-color: var(--light-orange);
+  padding-top: 10px;
+  align-content: center;
+}
+
+.social-icons {
+  display: flex;
+  flex-direction: column;
+}
+
+.social-icons img{
+  width: 100px;
+  height: auto;
+  padding-left: 1rem;
+  padding-right: 2rem;
+}
+
+.copyright-text h3{
+  text-align: center;
+}
+
+/* breakpoint 1 --> 540px */
+
+@media screen and (min-width: 540px) {
+  .hamburger {
+    display: none;
+  }
+
+  .hamburger-motto {
+  justify-content: flex-end;
+ 
+  }
+  .welcome {
+    grid-template-areas:
```

Comment:
remember you only need to redefine properties in media queries if they change. The values for `grid-template-areas` haven't changed so you don't need to add this property in the media query